modules/ROOT/pages/business-groups.adoc

```diff
@@ -104,7 +104,29 @@ Only an organization administrator that belongs to a business group can delete i
 . In the *Business Group Info* dialog, click *Delete Business Group*.
 . In the confirmation dialog, enter the name of the business, then click *Delete*.
 
-=== See Also 
+== Redistribute Resources between Existing Business Groups
+
+You may need to redistribute resources (such as vCores, VPCs, load balancers, and so on) based on deployment of apps in your business groups.
+The business group structure creates a hierarchical distribution of resources that prevents child business groups from consuming resources that the master organization needs to distribute elsewhere or use itself.
+As such, you can distribute resources from your master organization to business groups and then to their child business groups.
+The business group that receives resources can either consume them or distribute them to child business groups for consumption or further delegation.
+
+A child business group can only receive resources that are available in the parent business group that is directly above it in the hierarchy.
+For example, a child business group cannot receive sandbox vCores if the parent business group does not have any sandbox vCores available, even if the master organization has sandbox vCores available.
+Additionally, the master organization or parent business group cannot reclaim resources that are currently allocated to a business group that is lower in the hierarchy.
+
+You must have the Organization Administrator permission to redistribute resources.
+
+. In the Anypoint Platform *Access Management* page, click *Organization*.
+. Click the name of existing business group. +
+The *Business Group Info* window appears, showing the business group settings.
+. Enter the number of vCores you want the business group to have.
+. Click *Save changes*.
+
```

Comment:
Do we need the following description of what the UI shows after your reassign resources?
Seems to me that the conclusion of the topic is reached right here.

Reply:
It doesn't change beyond the sliders and numbers they adjusted.